src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,113 @@
+.cards {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+
+  &__collection {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: ($card-width * $cards-in-row) +
+      ($space-between-cards * ($cards-in-row - 1));
+  }
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $space-between-cards;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    display: block;
+    font-size: 12px;
```

Comment:
`font-size: 12px;` is in `page.scss`, check all code, I think there is no matter to dublicate it

Reply:
agree